.github/workflows/voter.yml

```diff
@@ -41,6 +41,7 @@ jobs:
 
     - name: Build and analyze
       env:
+        RUNNING_IN_GITHUB: "true" # used to disable tests that are extremely influenced by available CPU time
```

Comment:
I guess you can rely on a env variable already provided by GA: https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables

```suggestion
```